app/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,30 @@
+require "gds_api/link_checker_api"
+require "local-links-manager/check_links/link_status_updater"
+
+class WebhooksController < ApplicationController
+  skip_before_action :require_signin_permission!
+  skip_before_action :verify_authenticity_token
+  before_action :verify_signature
+
+  def link_check_callback
+    LocalLinksManager::CheckLinks::LinkStatusUpdater.new.call(
+      GdsApi::LinkCheckerApi::BatchReport.new(
+        params.to_unsafe_hash
+      )
+    )
+  end
+
+  def verify_signature
+    given_signature = request.env["HTTP_X_LINKCHECKERAPI_SIGNATURE"]
```

Comment:
can we use a request.headers method? looking at [source](https://github.com/rails/rails/blob/d2dde6cbfc11bf39be30d22e2ee7c41c119e0d51/actionpack/lib/action_dispatch/http/request.rb#L188) they header might be formatted in header spec there (or maybe it's a nightmare because they format them different)